common/index.html

```diff
@@ -89,7 +63,10 @@ <h2>Your Browser is Out of Date</h2>
   </div>
 
   <script>
-    (function() {
+    (function () {
+      // check where the file is running
+      __webpack_public_path__ = window.location.protocol.startsWith('file') ? './' : '/'
```

Comment:
I don't see this used anywhere else, is this necessary?

Reply:
Ah, for context it looks like this was the intent: https://webpack.js.org/guides/public-path/#on-the-fly. I don't think this works though because this variable is only picked up on at compile time, but this js only runs on page load time.

This has the problem of when a page is run on a webserver (i.e. production) that any page with 2 levels of nesting will not load. I tested this by running the build through superstatic, a simple node webserver. As you can see in the screenshot, it loaded the assets at `/account/*` instead of the root level. Webservers need assets at `/`, where as the downloadable version needs them at `./` or no slash at all.

<img width="660" alt="screen shot 2017-12-24 at 4 26 18 pm" src="https://user-images.githubusercontent.com/649992/34329271-5c9342ca-e8c7-11e7-820f-397116ee1edc.png">

Reply:
tl;dr I think the only reasonable way to do this is have a separate build for the downloadable version.